content/self-host/rustdesk-server-pro/smtp/microsoft-365/_index.de.md

```
@@ -22,7 +22,7 @@ Allgemeine SMTP-Einstellungen finden Sie unter [SMTP](../).
 | OAuth2 Client secret | Der unter `Certificates & secrets` erstellte Secret-`Value` |
 
 Dieser Screenshot zeigt, wo diese Werte in RustDesk eingetragen werden:
-![](/docs/en/self-host/rustdesk-server-pro/smtp/Microsoft365/images/21-RustDesk-SMTP-OAuth2-2.png)
+![](/docs/en/self-host/rustdesk-server-pro/smtp/microsoft-365/images/21-RustDesk-SMTP-OAuth2-2.png)
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)